app/models/timdex_record.rb

```diff
@@ -69,6 +69,7 @@ class TimdexRecord < TimdexBase
         }
         numbering
         physicalDescription
+        provider
```

Comment:
Yes, that's correct. It sucks, but I suspect that refactoring the logic will suck more. 🙂 Refactoring everything into fragments seems a bit tricky, and while I think it's a good idea, it feels very post-MVP to me given the current timeline. In fact, I think part of that change would involve a spike ticket evaluating whether we even want to use the `graphql-client` gem at all, since we're not really using any of its features.